src/data/faq.json

```
@@ -1584,16 +1609,19 @@
                                     "The processing time of a PCR test (from taking the sample at your doctor's office or test center to the notification in the app) can be up to 48 hours. If you scanned your PCR-Test QR code into the App after a Corona test and did not receive a test result via the App within 3 days of making the test, it is possible that",
                                     "<ul><li>the respective laboratory is not fully linked with the Corona-Warn-App backend.</li><li>the consent field allowing the transmission of the test result to the Corona-Warn-App was not ticked on the submission form (<a href='/assets/documents/MU_10C_(01_2021)_web.pdf' target='_blank' >10C</a>/<a href='/assets/documents/MU_OEGD_(08_2021)_web.pdf' target='_blank' >OEGD</a>) sent by your doctor to the laboratory.</li></ul>",
                                     "Please check whether the laboratory working with your doctor or test center is connected to the Corona-Warn-App and whether the consent field is checked on the form. Only then is it ensured that the procedure used works with the Corona-Warn-App.",
                                     "Currently, 90% of the existing test capacities in Germany are covered by the complete digital process. We are continuing to work with the laboratories to close the gaps and to connect and upgrade all laboratories for the process. Therefore it can happen that a laboratory gets the order for the test with the correct form, but has to transmit the test result to the tested person in another way.",
-                                    "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a teleTAN from the TAN Hotline (<a href='#international_phone_numbers' target='_blank' >click here to see our phone numbers</a>) to warn your contacts via the Corona-Warn-App. Before you are able to enter the teleTAN into the app, you have to delete the pending test result from the app. To do this, please tap on",
+                                    "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a teleTAN from the TAN Hotline (<a href='#international_phone_numbers' target='_blank' >click here to see our phone numbers</a>) to warn your contacts via the Corona-Warn-App. Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ?????h. Before you are able to enter the teleTAN into the app (version 2.28 or smaller), you have to delete the pending test result from the app. To do this, please tap on",

```

Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "In this case, please call the responsible medical staff or test center/health authority to get your test result. If you are informed of a positive test result, you can request a teleTAN from the TAN Hotline (<a href='#international_phone_numbers' target='_blank' >click here to see our phone numbers</a>) to warn your contacts via the Corona-Warn-App. Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h. Before you are able to enter the teleTAN into the app (version 2.28 or smaller), you have to delete the pending test result from the app. To do this, please tap on",
```